deployment/helm/balloons/README.md

```diff
@@ -98,6 +98,7 @@ customize with their own values, along with the default values.
 | `hostPort`               | 8891                                                                                                                          | metrics port to expose on the host                   |
 | `config`                 | see [helm chart values](tree:/deployment/helm/balloons/values.yaml) for the default configuration                       | plugin configuration data                            |
 | `nri.patchRuntimeConfig` | false                                                                                                                         | enable NRI in containerd or CRI-O                    |
+| `nri.setPodPriorityClassNodeCritical` | false                                                                                                                         | enable [marking Pod as node critical](https://kubernetes.io/docs/tasks/administer-cluster/guaranteed-scheduling-critical-addon-pods/#marking-pod-as-critical)                       |
```

Comment:
Accompaniying my other comment:

```suggestion
| `podPriorityClassNodeCritical` | false                                                                                                                         | enable [marking Pod as node critical](https://kubernetes.io/docs/tasks/administer-cluster/guaranteed-scheduling-critical-addon-pods/#marking-pod-as-critical)                       |
```

...and ditto all charts

Reply:
@marquiz I have updated the helm charts as suggested. I've added the description of the "new" value `podPriorityClassNodeCritical` at the end of the helm value table, reflecting its position in the `values.yaml` files. I am resolving this thread in favour of the first thread.